events/bingo/entityScripts/scannerZone/bingoScannerZone.js

```diff
@@ -0,0 +1,48 @@
+//
+// bingoMachineZone.js
```

Comment:
wrong filename (please check in other files too, i might have missed commenting on some)